### PR TITLE
[10.0][ADD] add new module to display discount value on sale orders

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
This addon allows to see the discount value in the total section of a sale order.

Solve #694

### CODE REVIEW

**Comment by Cedric-Pigeon** [comment]
@lmignon @sebastienbeau

**Comment by pedrobaeza** [comment]
Shouldn't it be `sale_discount_display_amount`?

**Comment by Cedric-Pigeon** [comment]
@pedrobaeza Renamed.

File: sale_discount_display_amount/README.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Sale Discount Display Amount
+============================
+
+This addon intends to display the amount of the discount computed on
+sale_order_line and sale_order level
```
Comment (sbidoul) [inline sale_discount_display_amount/README.rst]: Can you elaborate the use case a little bit, and use the new readme fragments?

**Comment by Cedric-Pigeon** [comment]
@sbidoul Done.

**Review by sbidoul — APPROVED** [review]

File: sale_discount_display_amount/models/sale.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                line.order_id.currency_id,
+                line.product_uom_qty,
+                product=line.product_id,
+                partner=line.order_id.partner_shipping_id)
+
+            price_total_no_discount = taxes['total_included']
+            discount_total = price_total_no_discount - line.price_total
+
+            line.update({
+                'discount_total': discount_total * -1,
```
Comment (lmignon) [inline sale_discount_display_amount/models/sale.py]: @Cedric-Pigeon Discount must be absolute. (A negative discount is a surcharge)

File: sale_discount_display_amount/models/sale.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
```
Comment (lmignon) [inline sale_discount_display_amount/models/sale.py]: The 2 models should be in separate files. sale_order.py and sale_order_line.py

**Review by lmignon — CHANGES_REQUESTED** [review]

**Comment by Cedric-Pigeon** [comment]
@lmignon Done. Thanks

**Review by lmignon — APPROVED** [review]

**Review by sebastienbeau — APPROVED** [review]
Ok for me, thanks for the great job

**Pull request merged**